### PR TITLE
percona-toolkit 3.2.1

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
license ref:

```
License:

    This software is dual licensed, either GPL version 2 or Artistic License.

    This package is free software; you can redistribute it and/or modify
    it under the terms of the GNU General Public License as published by
    the Free Software Foundation, version 2; OR the Perl Artistic License.

    This package is distributed in the hope that it will be useful,
    but WITHOUT ANY WARRANTY; without even the implied warranty of
    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
    GNU General Public License for more details.

    You should have received a copy of the GNU General Public License
    along with this package; if not, write to the Free Software
    Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA

On Debian systems, the complete text of the GNU General Public License version
2 can be found in `/usr/share/common-licenses/GPL-2'.

On Debian systems, the complete text of the Artistic License can  be found in
`/usr/share/common-licenses/Artistic'.
```

File: Formula/percona-toolkit.rb [hunk 1]
```diff
@@ -5,9 +5,9 @@ class PerconaToolkit < Formula
 
   desc "Percona Toolkit for MySQL"
   homepage "https://www.percona.com/software/percona-toolkit/"
-  url "https://www.percona.com/downloads/percona-toolkit/3.2.0/source/tarball/percona-toolkit-3.2.0.tar.gz"
-  sha256 "e7c7028631d5b1c66ba21841c55919537bd4ed2f3a5975f989fa4abd7d9d753d"
-  license "GPL-2.0"
+  url "https://www.percona.com/downloads/percona-toolkit/3.2.1/source/tarball/percona-toolkit-3.2.1.tar.gz"
+  sha256 "92da0d3712533205f283515ff47ce68521fc2de20989c45b4c65a9743d0f7487"
+  license any_of: ["GPL-2.0-only", "Artistic-1.0-Perl"]
```
Comment (Rylan12) [inline Formula/percona-toolkit.rb:10]: Why `Artistic-1.0-Perl`?

Did it specify a version for the artistic license? If not, should we go with `Artistic-1.0+` to allow any artistic license?

Comment (chenrui333) [reply Formula/percona-toolkit.rb:10]:
```
On Debian systems, the complete text of the Artistic License can  be found in
`/usr/share/common-licenses/Artistic'.
```

On debian systems, the common license is the perl edition.

Comment (Rylan12) [reply Formula/percona-toolkit.rb:10]: Ah! Didn't realize that!

**Review by Rylan12 — APPROVED** [review]

**Pull request closed without merging**